src/forms/NewStudyForm.js

```diff
@@ -1,17 +1,27 @@
 import React, {useState, Fragment} from 'react';
 import {Switch, Route} from 'react-router-dom';
 import PropTypes from 'prop-types';
-import {Form, Segment, Message, Step} from 'semantic-ui-react';
+import {Form, Segment, Message, Step, Button, Header} from 'semantic-ui-react';
 import {Formik} from 'formik';
-import {InfoStep, ExternalStep, GrantStep} from './StudyFormSteps';
+import {InfoStep, ExternalStep, LogisticsStep} from './StudyFormSteps';
 
-const NewStudyForm = ({submitValue, apiErrors, history}) => {
+const NewStudyForm = ({
+  submitValue,
+  apiErrors,
+  prefillValues,
+  newStudy,
```

Comment:
can we try computing this inside the component instead of passing it as a prop? This way the component will be more encapsulated 

```js
const initialValues = prefillValues
    ? prefillValues
    : {
        externalId: null,
        name: null,
        shortName: null,
        description: null,
        releaseDate: null,
        version: null,
        attribution: null,
        anticipatedSamples: 0,
        awardeeOrganization: null,
      };

const newStudy = Object.values(intiialValues).some(val => Boolean(val) === true);
```